### DISCUSSION TITLE
Highlighted Card

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1990] [Proposals]
### Goal

Avaliar a proposta da adição de um _pretitle_  para a melhoria do componente Highlighted Card.

### Context

Quando olhamos para os outros Cards da biblioteca Mística, em sua grande maioria, usamos o elemento de texto _pretitle_, com isso, estamos solicitando essa melhoria para o componente em questão visando uma aplicação mais consistente e padronizada.

### Impact

Adicionando o _pretitle_ teríamos mais consistência com os demais Cards trazendo a possibilidade de introduzir o nome do produto e/ou serviço, como fazemos com os Cards aplicados nas jornadas do App Vivo.

### Discussion Questions

- Seria possível adicionar o _pretile_ no componente?
- Poderíamos prever essa melhoria nos needs para Q1?

### Do you have a visual proposal?

https://www.figma.com/design/4VAHJixmh4gRLGv4REbWih/POC-Template-(Copy)?node-id=4701-3165&t=rqKPuKlcr7U6hsdc-1

**Comment by AnaMontes11** [comment]
Hi @annefranca, from our side, we agree to include the _pretitle_ in the Highlighted Card. Please include it in your 'Needs' kanban board and prioritize it based on its level of importance.
Thanks!

**Reply by annefranca** [reply]
Ticket: https://jira.tid.es/browse/OBVIVO-2889

**Comment by yceballost** [comment]
https://github.com/Telefonica/mistica-design/issues/2061
